vmaas_sync/vmaas_sync.go

```diff
@@ -0,0 +1,54 @@
+package vmaas_sync
+
+import (
+	"app/base/utils"
+	"github.com/gin-gonic/gin"
+	"github.com/gorilla/websocket"
+	ginprometheus "github.com/zsais/go-gin-prometheus"
+)
+
+func runMetrics() {
+	// create web app
+	app := gin.New()
+
+	prometheus := ginprometheus.NewPrometheus("gin")
+	prometheus.Use(app)
+	err := app.Run(":8083")
+	if err != nil {
+		utils.Log("err", err.Error()).Error()
+		panic(err)
+	}
+}
+
+func runWebsocket(conn *websocket.Conn, handler func(data []byte, conn *websocket.Conn)) {
+	defer conn.Close()
+
+	for {
+		typ, msg, err := conn.ReadMessage()
```

Comment:
service also needs to write to websocket `subscribe-listener` message to receive notifications

see https://github.com/RedHatInsights/vulnerability-engine/blob/master/vmaas_sync/vmaas_sync.py#L299

Reply:
Thanks, fixed.